### ISSUE TITLE
Canonizer Shares algorithm - Latest month share value is not showing 

### ISSUE THREAD

**Issue by rupalichavan9860** [the-canonizer/Canonizer-Beta--Issue-Tracking#1388]
Latest share value for "Rupali C" - is "8652"
![image](https://github.com/user-attachments/assets/96f8cfe0-6f3b-4ac4-ad9b-a25b45324366)

But it is showing share value of month Aug 2024  
![image](https://github.com/user-attachments/assets/0319317e-6fef-4901-9c72-38f0692fc04b)

**Comment by rupalichavan9860** [comment]
This is working fine now so closing this

**Comment by sajid315** [comment]
> This is working fine now so closing this

Yes @rupalichavan9860 , I also checked the same for my local and ux-dev server. Moving the ticket ahead.

**Comment by rupalichavan9860** [comment]
Working fine now

**Comment by rupalichavan9860** [comment]
Working fine on live
